src/con.c

```diff
@@ -1970,6 +1975,31 @@ void con_toggle_layout(Con *con, const char *toggle_mode) {
     }
 }
 
+/**
+ * This function changes the way new containers get added to layouts. The
+ * 'default' means the layout is filled left-to-right or top-to-bottom
+ * depending on orientation. 'reverse' changes that to right-to-left or
+ * bottom-to-top. 'toggle' inverts the setting depending on its previous value.
+ *
+ */
+void con_set_layout_fill_order(Con *con, const char *fill_order) {
+    Con *parent = con;
+    /* Users can focus workspaces, but not any higher in the hierarchy.
+     * Focus on the workspace is a special case, since in every other case, the
+     * user means "change the layout of the parent split container". */
+    if (con->type != CT_WORKSPACE)
+        parent = con->parent;
+    DLOG("con_set_fill_order(%p, %s), parent = %p\n", con, fill_order, parent);
+
+    if (strcasecmp(fill_order, "default") == 0) {
```

Comment:
Maybe also make the function accept `NULL`?

```suggestion
    if (fill_order == NULL || strcasecmp(fill_order, "default") == 0) {
```

Reply:
Any further update on this PR? @wlhlm are you still with us :pray: ? Tbh this glaring missing feature of I3 is the ONLY thing that keeps me (and I'm very sure many others) from switching from Bspwm over to I3. Can some good soul make the extra effort to push this through the finish line?